### ISSUE TITLE
AIS Locations not despawning as expected

### ISSUE THREAD

**Issue by Will-Nichols** [fencr0c/fen_a3#22]
I posted about this on the Steam page discussion but thought you might want to see the image I captured in the game. This was on our dedicated server earlier today. If you notice my camera as Zeus and player were located just south of Katkoula (Tanoa map) and the red enemy icons all around the map and the civilians as well were all synced to the various AIS Location modules in their areas as well.
[https://steamcommunity.com/sharedfiles/filedetails/?id=1567541739](url)

**Comment by fencr0c** [comment]
From what you’ve written above, it sounds like you’re sync’ing the AI groups to the AIS Location module, that’s not correct. The AI groups should be sync’d to an AIS Group module. Despawn may take up to 5 minutes to kick in, and only AI groups and objects assigned to the AIS Location and within the AIS activation radius are despawned. Vehicles that have sustained damage will not despawn. Have a look at the AIS Quick Start Video, links to YouTube on are on the Steam Workshop page.

**Comment by Will-Nichols** [comment]
Going to stop posting this on the Steam Workshop page discussion. Ran the test last night; some items still are not despawning I've resynchronized  all AI units where needed using the AIS group module.

Is there a limit on the number of AI that can be synced up like that? Getting several that are not despawning. When they do come out on respawning; they do not follow the way points they are assigned or they do until one certain point and then they freeze.

I I'm going to go back in and look to see which ones are synced up that are not despawning. A few that I looked at last night my time are units that are either on a turret or in vehicles and I may not have selected the commander of said vehicle. But I also noticed that most of the time when a vehicle spawns in they are exploding.
